sites/org/next.config.mjs

```diff
@@ -6,6 +6,20 @@ import { jargon } from './jargon.mjs'
 
 let config = configBuilder({ site: 'org', jargon })
 config.i18n = i18nConfig.i18n
+config.redirects = async () => {
```

Comment:
Can we use [a rewrite](https://nextjs.org/docs/pages/api-reference/next-config-js/rewrites) for this.
I'd prefer to keep the URL clean for page 1.